src/routes/getD3Data.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+require('env2')('./config.env');
+const get = require('./../database/db_get');
+
+const app = express.Router();
+
+module.exports = [
+  app.get('/getd3data', (req, result) => {
```

Comment:
This is not a good endpoint name

better:

`GET /memories?userId={userId}`

You do not have to use the userId, but your endpoints keep the same format thought development